CCSA_ChatApp.Authentication/Services/AuthenticationService.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.IdentityModel.Tokens;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CCSA_ChatApp.Authentication.Services
+{
+    public class AuthenticationService
+    {
+        
+        public static void ConfigureServices(IConfiguration config,IServiceCollection service)
+        {
+            IConfiguration _jwtKey = config.GetSection("JWTSettings");
+            service.AddAuthentication(JwtBearerDefaults.AuthenticationScheme).
+            AddJwtBearer(options =>
+            {
+                options.TokenValidationParameters = new TokenValidationParameters
+                {
+                    ValidateIssuer = false,
+                    ValidateAudience = false,
+                    ValidateIssuerSigningKey = true,
+                    ValidIssuer = _jwtKey.GetSection("validIssuer").Value,
+                    ValidAudience = _jwtKey.GetSection("validAudience").Value,
```

Comment:
I didn't see the validAudience in appsettings.json, where did you get it from? Also the "IssuerName" that I saw there for  validIssuer is it a placeholder?

Reply:
Thanks for pointing that out for me. I removed the validAudience.

Reply:
Okay no problem